### ISSUE TITLE
Console is stuck in "Running" state

### ISSUE THREAD

**Issue by ocafebabe** [ansible-semaphore/semaphore#870]
I'm experiencing an issue with Semaphore where the console is stuck in "_Running_" state indefinitely even though the playbook has failed (exit code 1): 
![Screenshot from 2022-02-22 13-05-06](https://user-images.githubusercontent.com/3409735/155193404-b1fe530f-bdfc-4578-865f-1958ada374be.png)

This is happening only when the playbook is generating a lot of output (+1000 lines) on an error and it seems to be similar to this issue: #642 

For reference, I'm using the latest released version: 2.8.53 and it's a major show stopper for us.

**Comment by ocafebabe** [comment]
I did another test with the same playbook to reproduce the same behaviour and I discovered something interesting, while the console is still showing a "**Running**" state, the task is correctly shown with a "**Failed**" status under _Task Templates:
![Screenshot from 2022-02-22 13-41-50](https://user-images.githubusercontent.com/3409735/155214101-205add6d-e351-4f29-89bb-27f4b6cb2e7a.png)

**Comment by fiftin** [comment]
Thank you, I will try to reproduce the issue.

**Comment by ocafebabe** [comment]
Thanks a lot and please let me know if you need additional details or testing from my part.

**Comment by wdsff1** [comment]
Unfortunately I have the same problem. When it happens, the console output is "Enter passphrase for key '/ansible/semaphore/access_key_..'

**Comment by steadfasterX** [comment]
My issue is not 100% the same but the browser hangs up especially when checking "Debug" checkbox and/or using -vvv / -vvvv as arg. This is not an issue when not using Debug or not using -vvv args. it just happens when a LOT of output is coming in. 
when I am able to stop the task (it keeps running forever here) it stays in "stopping" state forever, blocking any other tasks ofc (no parallel runs allowed)..

**Comment by piotrszczypta-a4bee** [comment]
I got the same error. Ansible semaphore version 2.9.37. It happens only in debug mode and when output is really long.
![Bez tytułu](https://github.com/ansible-semaphore/semaphore/assets/89578086/905694be-14e9-4b03-a8d4-f0462ad9cc60)


However, in task history it is showed as ended
![image](https://github.com/ansible-semaphore/semaphore/assets/89578086/80cab890-25f5-4ac5-8b0b-0dc8ecb96116)

**Comment by openstreem** [comment]
Hello...has there been any further investigation of this bug? I'm seeing similar behavior. Play seems to hang with > 1000 lines of output, however, history will show that the Play was successful.

**Comment by cheretbe** [comment]
Same here. Easy to reproduce: setting debug (-vvvv) or diff (--diff) options on almost every playbook makes the job to output clipped results and then hang in "running" state forever. Direct connection to v2.12.14 docker service, no reverse proxy.

**Comment by fiftin** [comment]
Hi @cheretbe what is latest line in the task log?

I have `Escalation succeeded`.

**Comment by cheretbe** [comment]
@fiftin Hi, nothing specific, just current state of the playbook output
For example: 
![Image](https://github.com/user-attachments/assets/68b05e4b-6f96-42ea-88dd-d73274e3b439)
Server log at this point doesn't contain anything unusual
```
semaphoreui        | time="2025-02-24T07:15:13Z" level=info msg="Task 31 added to queue"
semaphoreui        | time="2025-02-24T07:15:13Z" level=info msg="Set resource locker with TaskRunner 31"
semaphoreui        | time="2025-02-24T07:15:13Z" level=info msg="Task 31 removed from queue"
semaphoreui        | time="2025-02-24T07:15:14Z" level=warning msg="read |0: file already closed" error="Failed to read TaskRunner output"
semaphoreui        | time="2025-02-24T07:15:14Z" level=warning msg="read |0: file already closed" error="Failed to read TaskRunner output"
semaphoreui        | time="2025-02-24T07:15:14Z" level=warning msg="read |0: file already closed" error="Failed to read TaskRunner output"
semaphoreui        | time="2025-02-24T07:15:18Z" level=warning msg="bufio.Scanner: token too long" error="Failed to read TaskRunner output"
```

**Comment by fiftin** [comment]
@cheretbe Did task complete? Looks like it is known websocket issue. I researched a lot of this issue but I didn't find the source and solution.

**Comment by cheretbe** [comment]
@fiftin No, it just stayed in "running" state for about an hour until I manually stopped it (normally it takes 30-40 seconds to complete). Is there anything I can do to help you pinpoint the issue? Like enabling verbose output somewhere.
It doesn't affect my workflow right now, but it would be nice to get rid of this bug.

**Comment by cheretbe** [comment]
@fiftin Fwiw I switched back to nginx reverse proxy and noticed a lot of error messages like this in the log:
```
semaphoreui        | time="2025-02-24T17:37:22Z" level=error msg="write tcp 172.24.0.3:3000->192.168.10.22:45394: use of closed network connection" error="Cannot send close message"
semaphoreui        | time="2025-02-24T17:38:25Z" level=error msg="websocket: close 1006 (abnormal closure): unexpected EOF" fields.level=Error
semaphoreui        | time="2025-02-24T17:38:25Z" level=error msg="write tcp 172.24.0.3:3000->192.168.10.22:52812: use of closed network connection" error="Cannot send close message"
```
192.168.10.22 - is the address of nginx proxy
